internal/common/foraging/deerhunt_test.go

```diff
@@ -0,0 +1,61 @@
+package foraging
+
+import (
+	"fmt"
+	"math"
+	"testing"
+
+	"github.com/SOMAS2020/SOMAS2020/internal/common/shared"
+)
+
+var huntParticipants = map[shared.ClientID]float64{shared.Team1: 1.0, shared.Team2: 0.9} // arbitrarily chosen for test
+var params = DeerHuntParams{P: 0.95, Lam: 1.0}
+var hunt = DeerHunt{Participants: huntParticipants, Params: params}
+
+func TestDeerUtilityTier(t *testing.T) {
+
+	decay := 0.8 // use an arbitrary decay param
+	maxDeer := 4
+
+	var tests = []struct {
+		inputR float64 // cumulative resource input from hunt participants
+		want   int     // output tier
```

Comment:
```suggestion
		want   uint     // output tier
```

Reply:
comments above variables btw

Reply:
Ignore if it actually shows on intellisense, was just reminded that struct variable comments should be on the side